nsxt/data_source_nsxt_policy_qos_profile_test.go

```diff
@@ -72,20 +92,38 @@ func testAccDataSourceNsxtPolicyQosProfileDeleteByName(name string) error {
 	if err != nil {
 		return fmt.Errorf("Error during test client initialization: %v", err)
 	}
-	client := infra.NewDefaultQosProfilesClient(connector)
 
 	// Find the object by name
-	objList, err := client.List(nil, nil, nil, nil, nil)
-	if err != nil {
-		return handleListError("QosProfile", err)
-	}
-	for _, objInList := range objList.Results {
-		if *objInList.DisplayName == name {
-			err := client.Delete(*objInList.Id)
-			if err != nil {
-				return handleDeleteError("QosProfile", *objInList.Id, err)
+	if testAccIsGlobalManager() {
+		client := gm_infra.NewDefaultQosProfilesClient(connector)
```

Comment:
do you think we should replace this with search function as well?

Reply:
Not sure what you mean. There is not default QoS profile that we can use here, so forced to create it.

Reply:
Done